ui/app/components/variable-form.hbs

```diff
@@ -148,9 +137,20 @@
   {{/if}}
 
   <footer>
+    {{#unless (eq this.view "json")}}
```

Comment:
There's an ember idiomatic style error here in order to prevent double-negative etc. template logic. Straightforward fix is to `@alias` `this.viewIsJson` to this equality.